src/engine/TileManager.cxx

```diff
@@ -24,6 +25,19 @@ TileData *TileManager::getTileData(const std::string &id) noexcept
   return nullptr;
 }
 
+std::vector<std::string> TileManager::getTileIDsOfCategory(Zones zone)
```

Comment:
I think not, better to hide data structure, but query for data you want. Should be fine place for this function. I suppose later this can/should be more general.

Reply:
The TileManager is responsible for storing and supplying TileData and tileIDs. that's why I put it there.